packages/wonder-blocks-dropdown/src/components/dropdown-core.js

```diff
@@ -748,10 +748,13 @@ class DropdownCore extends React.Component<Props, State> {
                     }}
                 >
                     {({placement, ref, style, outOfBoundaries}) => {
+                        // For some reason popper.js includes `pointerEvents: "none"` in the `style`
+                        // it passes to us, but only when running the tests.
+                        const {pointerEvents: _, ...restStyle} = style;
```

Comment:
To add some reasoning - it's possibly our mock PopperJS that's setting that style. When it was just used for Tooltips, this made sense as generally, we wouldn't want pointer events going to the tooltip, but instead to what's under it (we've changed hit detection since then, so this isn't "necessary" anymore, I don't think). My guess is that our mock of PopperJS is still adding this style.

Reply:
I'll have a look at the popper mock to see if that's what's happening.

Reply:
I looked at all of the __mocks__/popper.js.js files and I don't see anything in them that is mocking the style that's being returned.  I did some digging and found that our version of react-popper sets
```
  var initialStyle = {
    position: 'absolute',
    top: 0,
    left: 0,
    opacity: 0,
    pointerEvents: 'none'
  };
```
and defines a `getPopperStyle` helper like so
```
      _defineProperty(_assertThisInitialized(_this), "getPopperStyle", function () {
        return !_this.popperNode || !_this.state.data ? initialStyle : _extends({
          position: _this.state.data.offsets.popper.position
        }, _this.state.data.styles);
      });
```
I'll update the comment to say "react-popper" instead of "popper.js".